### DISCUSSION TITLE
Please help me to tell your favorite musician!!

### DISCUSSION THREAD

**Discussion by Iwauchi-ux** [NAIST-SE/AttractingContributors#63] [General]
Please help me to tell your favorite musician!!

https://github.com/Iwauchi-ux/AttractingContributors-Music/blob/main/README.md

**Comment by vittunyuta-aom** [comment]
added please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Comment by eedy8888** [comment]
added please help me.
https://github.com/eedy8888/AttractingContributors

**Comment by sergiodl-naist** [comment]
I added one that I have been listening to it a lot lately!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames
